src/integrations/analytics.ts

```diff
@@ -39,3 +39,31 @@ export async function sendAnalyticsEvent<T extends AnalyticsEvent>(
     { retries: 3 },
   );
 }
+
+export type ExperimentAnalyticsEvent = {
+  event_timestamp: Date;
+  user_id: string;
+  experiment_id: string;
+  variation_id: number;
+};
+
+export async function sendExperimentAnalyticsEvent<
```

Comment:
instead of experiment analytics, can we use the term experiment allocation please?

Reply:
Is it only allocation, thinking it might need something else down the line so just wanted to be abstract?

Reply:
The official term we use everywhere (including the table name in BigQuery) is experiment allocation. It's a single request with the experiment information that is stored in BQ and then we know how to split the user base